app/Http/Controllers/Assets/AssetsController.php

```diff
@@ -111,6 +111,13 @@ public function store(ImageUploadRequest $request)
         // differently
         $asset_tags = $request->input('asset_tags');
 
+        foreach($asset_tags as $asset_tag) {
+            if ($asset_tag > PHP_INT_MAX) {
```

Comment:
I think the only thing I might change here is to make this `>=` ? Because if you *do* try to save an asset with an asset tag of `PHP_INT_MAX` it's going to try to save that `+1` to the database - which will fail.